src/app/shared/plnkr/example/button-demo.html

```diff
@@ -0,0 +1,104 @@
+<div class="demo-button">
```

Comment:
I know these will eventually go in a separate npm package, but maybe for now put them in assets/examples?